_includes/components/posts-tags.njk

```diff
@@ -0,0 +1,10 @@
+<div class='h-10'>
```

Comment:
Dai I have created a snippet to render two tags above posts, but the tags are not related with the posts, they are the first two tags from the tagslist. Could not find how to gather tags related with the posts. Let me know if you have some idea. I will resume tomorrow.